sources/packages/backend/libs/services/src/sfas/sfas-individual.service.ts

```diff
@@ -94,28 +96,18 @@ export class SFASIndividualService {
 
   /**
    * Search for a record in SFAS table using student details.
-   * @param lastName student's last name.
-   * @param birthDate student's birth date.
-   * @param sin student's sin.
+   * @param studentId Student id.
    * @returns SFAS individual details.
    */
-  async getSFASOverawards(
-    lastName: string,
-    birthDate: string,
-    sin: string,
-  ): Promise<SFASIndividual> {
+  async getSFASOverawards(studentId: number): Promise<SFASIndividual> {
```

Comment:
This method may removed depending on the answer that we get from the business.
Related comment: https://github.com/bcgov/SIMS/pull/4369#discussion_r1964429507